### PR TITLE
Add shell scripting tutorial

### PR DESCRIPTION
**Pull request by debajyotig466**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [x] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [x] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [x] Include author(s) and platform where appropriate.
- [x] Put lists in alphabetical order, correct spacing.
- [x] Add needed indications (PDF, access notes, under construction).
- [x] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by eshellman** [comment]
Be sure to check the free-programming-books-lint output for linter errors. Alphabetical order and spacing are the most common problems.

File: courses/free-courses-en.md [hunk 1]
```diff
@@ -206,6 +206,8 @@
 * [How to CMake Good](https://www.youtube.com/playlist?list=PLK6MXr8gasrGmIiSuVQXpfFuE1uPT615s) - vector-of-bool (YouTube)
 * [Shell Scripting Tutorial](https://www.youtube.com/playlist?list=PL7B7FA4E693D8E790) - The Bad Tutorials (YouTube)
 * [Shell Scripting Tutorial for Beginners](https://www.youtube.com/playlist?list=PLS1QulWo1RIYmaxcEqw5JhK3b-6rgdWO_) -  ProgrammingKnowledge (YouTube)
+* [Shell Scripting Tutorial](https://youtu.be/GtovwKDemnI) - edureka!(YouTube)
+
```
Comment (davorpa) [inline courses/free-courses-en.md:210]: Expand urls and fix whitespaces

```suggestion
* [Shell Scripting Tutorial](https://www.youtube.com/watch?v=GtovwKDemnI) - edureka!
```

**Review by davorpa — CHANGES_REQUESTED** [review]
Not a playlist but +1h video covering a topic

**Review by davorpa — APPROVED** [review]

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**